explorer/syntax/parser.ypp

```diff
@@ -491,7 +513,7 @@ unimpl_expression:
 ;
 value_expression:
   // ref_deref_expression excluded due to precedence diamond.
-  additive_expression
+  modulo_expression
 | as_expression
```

Comment:
```suggestion
  additive_expression
| modulo_expression
| as_expression
```

Per the [precedence diagram](https://github.com/carbon-language/carbon-lang/tree/trunk/docs/design/expressions#precedence), additive and modulo expressions are at the same precedence level, so it seems simpler to represent them as siblings in the grammar.